packages/ui-modal/src/Modal/__tests__/Modal.test.js

```diff
@@ -55,25 +57,33 @@ describe('<Modal />', async () => {
       </Modal>
     )
 
-    const modal = await ModalLocator.find(':label(Modal Dialog)')
-    const layout = await modal.find(`.${styles['fullscreenLayout']}`)
-
+    const layout = await ModalLocator.find(
+      '[class*="-modal__fullscreenLayout"]'
+    )
     expect(layout).to.exist()
   })
 
   it('should apply theme overrides when open', async () => {
-    await mount(
-      <Modal
-        open
-        size="small"
-        label="Modal Dialog"
-        shouldReturnFocus={false}
-        theme={{
+    const themeOverride = {
+      components: {
+        Modal: {
           smallMaxWidth: '10em'
-        }}
-      >
-        <Modal.Body>Foo Bar Baz</Modal.Body>
-      </Modal>
+        }
+      }
+    }
+    await mount(
+      <EmotionThemeProvider theme={canvas}>
+        <EmotionThemeProvider theme={themeOverride}>
+          <Modal
+            open
+            size="small"
+            label="Modal Dialog"
+            shouldReturnFocus={false}
+          >
+            <Modal.Body>Foo Bar Baz</Modal.Body>
+          </Modal>
+        </EmotionThemeProvider>
+      </EmotionThemeProvider>
```

Comment:
The second EmotionThemeProvider is not needed, you can simply add the override directly to the component on the `themeOverride` key.

```suggestion
      <EmotionThemeProvider theme={canvas}>
          <Modal
            open
            size="small"
            label="Modal Dialog"
            shouldReturnFocus={false}
            themeOverride={{ smallMaxWidth: '10em' }}
          >
            <Modal.Body>Foo Bar Baz</Modal.Body>
          </Modal>
      </EmotionThemeProvider>
```